src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-is-hover: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
```

Comment:
1. remember to set `box-sizing: border-box` for all elements, it's a good practice
2. I would better set font-weight  for all header elements of each link, not globally for body

Reply:
done!